src/TerminalFighter/TextArea/TextArea.h

```diff
@@ -0,0 +1,20 @@
+#include <string>
+#include <SDL_ttf.h>
+
+namespace{
+    const SDL_Color COLOR_WHITE = {255, 255, 255};
+    const SDL_Color COLOR_GRAY = {100, 100, 100};
+}
+
+class TextArea {
+public:
+    TextArea(std::string, SDL_Color, int, int, int, int, std::string, int);
+    ~TextArea();
+    void render(SDL_Renderer& renderer);
```

Comment:
Does everything else call these render?  They used `draw()` instead right?

Reply:
Not sure what you mean here?

Reply:
The other classes call it `draw()` instead of `render()` i think

Reply:
Oh I see, yea they're called draw. I'll make the change